user-guide/docs/usecases/lowes/usecase.md

```diff
@@ -1,16 +1,18 @@
+/// html | header
+
 ## Seismic Response of Concrete Walls
 
-<span style="font-size:1.5em;">
-**Modeling Reinforced Concrete Walls using Shell Elements in OpenSees and Using Jupyter to Post Process Results**
-</span>
+Modeling Reinforced Concrete Walls using Shell Elements in OpenSees and Using Jupyter to Post Process Results
+
+///
```

Comment:
**Why?** To make long title and short/nav title look coupled (and short/nav title less important).

<details>

Before me:

1. The navigation had been using short titles, and the pages had been using long titles.
2. The documents were concatenated and navigation changed to match TACC user experience.
3. That caused navigation titles to differ from page titles.
4. A solution was implemented to put long and short titles on the page, so:
    - User could see the short nav title on the page, to check navigation.
    - User could see the long title on the page, to know the full name.

After me:

5. TACC styles made the short title and long title look less coupled.
6. I changed the markup so I could add CSS to:
    - visually couple the tiles
    - make short title be less important

After this PR:

7. We will review reverting to the old navigation (step 1).

</details>